doc/logo.md

```diff
@@ -0,0 +1,177 @@
+https://github.com/PDP-10/its/blob/alexey/CLOGO/doc/_info_/clogo.manual
+
+### MIT LISP LOGO
+
+LISP LOGO is an implementation of the LOGO language written in MACLISP for the ITS, TEN50
+and TENEX PDP-10 systems, and MULTICS. The system were implemented by Ira Goldstein,
+Henry Lieberman in the early 70's at MIT.
+One of the reason for this implementation was to provide a natural transition to the more
+powerful computational world of LISP as the Logo programmer grows more sophisticated.
+When desired, the one has access to all of the capabilities of MACLISP.
+
+As far as we know there were several LISP LOGO versions:
+- LLOGO - Standard version which uses a vocabulary which is compatible with 11LOGO.
+- CLLOGO - A version which uses a vocabulary which is compatible with CLOGO.
+- NLLOGO - An experimental version of LISP LOGO.
+- ELLOGO - Another experimental version of LISP LOGO.
+- BWLOGO - LISP LOGO with black and white graphics.
+
+LISP LOGO had available several packages of special functions:
+- TV TURTLE - LISP LOGO package provides Knight TV system Logo Graphics.
+- DISPLAY TURTLE - LISP LOGO package provides 340 / GT40 display Logo Graphics.
+- GERMLAND - some kind of display turtle for character displays without grahics capabilities.
+- MUSIC BOX - the package provides Logo music capabilities.
+
+The LLOGO variant of LISP LOGO was recently brought back to life during several amazing
+hacking session performed by Eric Swenson and Lars Brinkhoff. As a result, we
+have LLOGO with TV TURTLE and DISPLAY TURTLE running on ITS under emulation.
+
+to run, type:
+`:llogo`
+
+If you start from a TV console, LLOGO will ask "DO YOU WANT TO USE THE TV TURTLE?".
+In other cases, LLOGO will ask the question without the "TV" part and then it'll use
+the Lisp display slave. The Lisp display slave has the option to use the PDP-6 or PDP-10
+driving the 340 display, or to use the GT40. We don't have GT40 display slave in place yet.
+
+To switch back and forth between LOGO and LISP top level loops type `LISP` from LOGO
+prompt and `(LOGO)` from MACLSIP prompt respectively.
+
+One more powerful feature of LISP LOGO is using the MACLISP compiler directly on LOGO
+source programs and obtain a substantial gain in efficiency, once the programs are
+thoroughly debugged.
+
+Documentation:
+
+[MIT A.I. Memo 307A "LLOGO: An Implementation of LOGO in LISP"](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=2ahUKEwjNkYz73JngAhXGkywKHRfMDuQQFjAAegQICRAC&url=https%3A%2F%2Fdspace.mit.edu%2Fbitstream%2Fhandle%2F1721.1%2F6221%2FAIM-307a.pdf&usg=AOvVaw1iwaMXNU6NgcYAFToTxKlY)
+
+[MIT A.I. Memo 361 "The TV Turtle. A Logo Graphics System for Raster Displays"](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwj05u7d35ngAhVNl4sKHb2vAIQQFjABegQICRAC&url=https%3A%2F%2Fdspace.mit.edu%2Fbitstream%2Fhandle%2F1721.1%2F5773%2FAIM-361.pdf%3Fsequence%3D2&usg=AOvVaw3reoCDZkG-ZAyhdEe2B4G_)
+
```

Comment:
These two links go through Google.  Please make the direct link to dspace.mit.edu.

Reply:
done.